tests/CMakeLists.txt

```diff
@@ -337,10 +337,10 @@ if(WITH_PYTHON)
     restarting/from_7.1.0_until_7.2.0/UpgradeAndBackupRestore-2.toml)
   add_fdb_test(
     TEST_FILES restarting/from_7.1.0_until_7.2.0/VersionVectorDisableRestart-1.toml
-    restarting/from_7.1.0_until_7.2.0/VersionVectorDisableRestart-2.toml )
+    restarting/from_7.1.0_until_7.2.0/VersionVectorDisableRestart-2.toml IGNORE)
```

Comment:
Any other issues that we will need to address in order to enable these?

Reply:
yes, but not in the PR (I wrote in the PR comments). We still have recovery problems of the sort we have discussed in the past. But we do not appear to see issues with private mutations, now.